doc/ChangeLog

```diff
@@ -6,6 +6,8 @@ version 27-git (????-??-??):
     (variable: phpui.event_user_required, default: not set) [interduo]
   - enhancement: allow to choose identity type for customer [ravvar]
   - improvement: 'invoices.hide_ssn' support for debit notes [chilan]
+  - improvement: introduce 'phpui.show_closed_events' config variable i.e. it allows
```

Comment:
a tu Panie Kulego?

Reply:
Zapomniałem :)  Zmieniłem też getConfig na checkConfig bo jak ustawię phpui.show_closed_events=false to getConfig i tak zwróci mi 'true'.

Reply:
do Wiki puściłeś?

Reply:
Zaraz puszczę.

Reply:
Poszło.

Reply:
Dzięki - nie było co prawda wg alfabetu, ale już poprawiłem.

Reply:
'Następną razą' będzie. 10q.